src/diagnosers/10-ip.py

```diff
@@ -176,7 +176,7 @@ def is_default_route(r):
         def ping(protocol, target):
             return (
                 os.system(
-                    "ping%s -c1 -W 3 %s >/dev/null 2>/dev/null"
+                    "ping%s -c1 %s >/dev/null 2>/dev/null"
```

Comment:
Heuarg I'm not sure about that one ... I believe it's important to have a timeout otherwise the program may get stuck here ? (Or maybe there's a default timeout anyway ?) I'm confused about `ping6` not having a `-W` option ... there's also `ping -[4|6]`, I'm not sure why that one wasn't used ... But if I remember correctly there may be several versions of the `ping` command hence why it's a mess @_@

Reply:
Sooo I'd be curious to know how you noticed that bug ? And on that corresponding machine, check if `ping -6` exists and wether or not it's the same as `ping6` or what are the differences / from what deb package it is from

Reply:
Uuuhh I can't reproduce it :shrug:
I will revert this commit